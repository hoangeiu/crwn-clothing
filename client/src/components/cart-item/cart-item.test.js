import React from "react";

import { CartItem } from "./cart-item.component";

describe("CartItem component", () => {
  const mockItem = {
    imageUrl: "www.test.com",
    price: 10,
    name: "hsts",
    quantity: 2,
  };

  it("should CartItem renders without error", () => {
    expect(<CartItem {...mockItem} />).toMatchSnapshot();
  });
});
