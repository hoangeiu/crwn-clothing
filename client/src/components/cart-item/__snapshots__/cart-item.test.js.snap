// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartItem component should CartItem renders without error 1`] = `
<CartItem
  imageUrl="www.test.com"
  name="hsts"
  price={10}
  quantity={2}
/>
`;
